pkg/services/rendering/rendering.go

```diff
@@ -167,15 +177,13 @@ func (rs *RenderingService) Run(ctx context.Context) error {
 		}
 	}
 
-	if rs.pluginAvailable(ctx) {
+	if rp, exists := rs.RendererPluginManager.Renderer(ctx); exists {
```

Comment:
I think this changes behavior slightly to now load on background service `Run` vs earlier where the plugin was already loaded here. Not sure if this matters :thinking:

Reply:
Yeah - I will let the rendering experts weigh in here. cc @AgnesToulet apologies I know you're no longer taking care of this stuff, but would appreciate some more eyes in case I fudge something. Feel free to just put me in touch with the right people ❤️